### PR TITLE
tiflash: Add label schedule for TiFlash.

### PR DESCRIPTION
**Pull request by LittleFall**
### Which TiDB version(s) do your changes apply to? (Required)

Add label schedule doc for TiFlash.

- [x] master (the latest development version)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

https://github.com/pingcap/tidb/issues/21217

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by solotzg** [comment]
lgtm

**Comment by LittleFall** [comment]
PTAL~ @TomShawn

**Comment by LittleFall** [comment]
@TomShawn Is there any suggestions to fix the lint? I have no idea about it...thanks

**Comment by TomShawn** [comment]
> @TomShawn Is there any suggestions to fix the lint? I have no idea about it...thanks

You can click the **Details** link right beside the CI error to see the error message.
image
This indicates that "link-must-start-with-slash". Otherwise, the links are invalid.

File: tiflash/use-tiflash.md [hunk 1]
```diff
@@ -77,6 +77,78 @@ SELECT * FROM information_schema.tiflash_replica WHERE TABLE_SCHEMA = '<db_name>
 * AVAILABLE 字段表示该表的 TiFlash 副本是否可用。1 代表可用，0 代表不可用。副本状态为可用之后就不再改变，如果通过 DDL 命令修改副本数则会重新计算同步进度。
 * PROGRESS 字段代表同步进度，在 0.0~1.0 之间，1 代表至少 1 个副本已经完成同步。
 
+### 可用区设置
+
+如果在配置副本时为了考虑容灾，需要将 TiFlash 的不同数据副本分布在多个数据中心，则可以按如下步骤进行配置：
```
Comment (TomShawn) [inline tiflash/use-tiflash.md:82]:
```suggestion
在配置副本时，如果为了考虑容灾，需要将 TiFlash 的不同数据副本分布到多个数据中心，则可以按如下步骤进行配置：
```

File: tiflash/use-tiflash.md [hunk 2]
```diff
@@ -77,6 +77,78 @@ SELECT * FROM information_schema.tiflash_replica WHERE TABLE_SCHEMA = '<db_name>
...
+
+    例如：
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    ALTER TABLE t SET TIFLASH REPLICA 2 LOCATION LABELS "zone";
+    ```
+
+3. 此时 pd 就会根据设置的 label 进行调度，将表 t 的两个副本分别调度到两个可用区中。我们可以通过监控或 pd-ctl 来验证这一点：
```
Comment (TomShawn) [inline tiflash/use-tiflash.md:115]:
```suggestion
3. 此时 PD 会根据设置的 label 进行调度，将表 `t` 的两个副本分别调度到两个可用区中。可以通过监控或 pd-ctl 来验证这一点：
```

File: tiflash/use-tiflash.md [hunk 3]
```diff
@@ -77,6 +77,78 @@ SELECT * FROM information_schema.tiflash_replica WHERE TABLE_SCHEMA = '<db_name>
...
+
+    ```sh
```
Comment (TomShawn) [inline tiflash/use-tiflash.md:117]:
```suggestion
    ```shell
```

File: tiflash/use-tiflash.md [hunk 4]
```diff
@@ -77,6 +77,78 @@ SELECT * FROM information_schema.tiflash_replica WHERE TABLE_SCHEMA = '<db_name>
...
+        "labels": [
+          { "key": "engine", "value": "tiflash"},
+          { "key": "zone", "value": "z2" }
+        ],
+        "region_count": 9,
+
+        ...
+    ```
+    
+关于使用 label 进行副本调度划分可用区的更多内容，可以参考：[通过拓扑 label 进行副本调度](schedule-replicas-by-topology-labels)，[同城多数据中心部署 TiDB](multi-data-centers-in-one-city-deployment) 与 [两地三中心部署](three-data-centers-in-two-cities-deployment)。
```
Comment (TomShawn) [inline tiflash/use-tiflash.md:150]:
```suggestion
关于使用 label 进行副本调度划分可用区的更多内容，可以参考[通过拓扑 label 进行副本调度](/schedule-replicas-by-topology-labels.md)，[同城多数据中心部署 TiDB](/multi-data-centers-in-one-city-deployment.md) 与[两地三中心部署](/three-data-centers-in-two-cities-deployment.md)。
```

**Review by LiSong0214 — APPROVED** [review]

**Comment by LittleFall** [comment]
> > @TomShawn Is there any suggestions to fix the lint? I have no idea about it...thanks
> 
> You can click the **Details** link right beside the CI error to see the error message.
> image
> This indicates that "link-must-start-with-slash". Otherwise, the links are invalid.

ok, I miss the `.md`...

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**